### PR TITLE
Fix some methods in libdal

### PR DESCRIPTION
**Pull request by sorbeee**

### CODE REVIEW

File: src/Server/libdal/Facade/DbFacade.h [hunk 1]
```diff
@@ -32,7 +32,7 @@ class DbFacade : public IDbFacade
 
 	void AddIncome(const Income& income) override;
 	std::optional<Income> GetIncomeById(const IdType& id) override;
-	std::optional<std::vector<Income>> GetAllIncomes(const IdType& user_id) override;
+    std::vector<Income> GetAllIncomes(const IdType& user_id) override;
```
Comment (VasylDobrovolskyi) [inline src/Server/libdal/Facade/DbFacade.h:35]: Something wrong with code formatting

File: src/Server/libdal/Facade/IDbFacade.h [hunk 2]
```diff
@@ -34,7 +34,7 @@ class IDbFacade
 
 	virtual void AddIncome(const Income& income) = 0;
 	virtual std::optional<Income> GetIncomeById(const IdType& id) = 0;
-	virtual std::optional<std::vector<Income>> GetAllIncomes(const IdType& user_id) = 0;
+    virtual std::vector<Income> GetAllIncomes(const IdType& user_id) = 0;
```
Comment (VasylDobrovolskyi) [inline src/Server/libdal/Facade/IDbFacade.h:37]: Something wrong with code formatting

File: src/Server/libdal/Repositories/IncomeRepository/IncomeRepository.cpp [hunk 3]
```diff
@@ -10,7 +10,7 @@ namespace
 	const std::string& AMOUNT = "amount";
 	const std::string& CATEGORY_ID = "categoryId";
 	const std::string& ADD_TIME = "addTime";
-	const std::string& EXPERATION_TIME = "expirationTime";
+    const std::string& EXPIRATION_TIME= "expirationTime";
```
Comment (VasylDobrovolskyi) [inline src/Server/libdal/Repositories/IncomeRepository/IncomeRepository.cpp:13]: Something wrong with code formatting

File: src/Server/libdal/Repositories/ListRepository/ListRepository.cpp [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 namespace
 {
-	const std::string& TABLE_NAME = "List";
+    const std::string& TABLE_NAME = "List";
```
Comment (VasylDobrovolskyi) [inline src/Server/libdal/Repositories/ListRepository/ListRepository.cpp:5]: Something wrong with code formatting

**Review by VasylDobrovolskyi — APPROVED** [review]

**Review by Denys-Bykov-0 — APPROVED** [review]

**Review by VasylDobrovolskyi — APPROVED** [review]

**Pull request merged**